github_rules/github_repo_collaborator_change.py

```diff
@@ -0,0 +1,20 @@
+def rule(event):
```

Comment:
Love this rule for your use of custom `title` and `severity` functions to capture two similar concepts in one rule! I always disliked writing 10 rules to detect 10 slightly different things but using the exact same rule logic basically.